docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,251 @@ sidebar_label: Wikidata
 
 ## Overview
 
+OpenRefine includes powerful ways to both pull data from Wikidata and add data to it. 
 
+OpenRefine’s connections to Wikidata were formerly an optional extension, but are now installed automatically with the downloadable package. The Wikidata extension can be removed manually by navigating to your OpenRefine installation folder, and then looking inside `webapp/extensions/` and deleting the `Wikidata` folder inside. 
 
-*   Former extension, now core [but you can remove it].
-*   Account required
+You do not need a Wikidata account to reconcile your local OpenRefine project to Wikidata. If you wish to upload your cleaned dataset to Wikidata, you will need to register an account, and authorize OpenRefine with that account. 
 
+The best source for information about how OpenRefine works with Wikidata is [on Wikidata itself, under Tools](https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine). This has tutorials, guidelines on editing, and spaces for discussion and help.
 
-## Features
+## Reconciling with Wikidata
 
+Wikidata is, at this time, the most powerful [reconciliation service](reconciling) for OpenRefine. [It supports](https://reconciliation-api.github.io/testbench/):
+*   A large number of potential types to reconcile against
+*   Previewing and viewing entities
+*   Suggesting entities, types, and properties
+*   Augmenting your dataset with more information pulled from Wikidata 
 
+You can find the documentation and further resources [here](https://wikidata.reconci.link/).
 
-*   Edit Wikidata schema
-*   Manage Wikidata account
-*   Import schema
-*   Export schema
-*   Uploading edits to Wikidata
-*   QuickStatements export
-*   Schema alignment
-*   Issue detection
+For the most part, Wikidata reconciliation behaves the same way other reconciliation processes do, but there are a few processes and features specific to Wikidata. 
 
+### Language settings
 
-## More resources
+You can install a version of the Wikidata reconciliation service that uses your language.
 
-[https://wdreconcile.toolforge.org/](https://wdreconcile.toolforge.org/) 
+First, you need your language code: this is the two-letter code in the domain name of your Wikipedia (for instance "fr" if your Wikipedia is [https://fr.wikipedia.org/wiki/](https://fr.wikipedia.org/wiki/)).
```

Comment:
Is there any other way of getting the language code? A list would be an easier reference point (and would avoid mentioning Wikipedia here which could be potentially confusing in the context of wikidata)